docs/contracts/index.md

```diff
@@ -67,6 +72,20 @@ Finally, you can set the `DEBUG_CONTRACTS` constant to `True` to enable debuggin
 .. autodata:: python_ta.contracts.DEBUG_CONTRACTS
 ```
 
+## CLI
+
+The `python_ta.contracts` CLI can execute a file as `__main__` with contracts enabled. This option
```

Comment:
The second sentence ("This option...") is redundant with the "limitations" section below. Remove this sentence.